src/MaterialDesignThemes.Wpf/TreeListView.cs

```diff
@@ -259,4 +262,68 @@ internal void MoveSelectionToParent(TreeListViewItem item)
             }
         }
     }
+
+    protected override void OnPropertyChanged(DependencyPropertyChangedEventArgs e)
+    {
+        base.OnPropertyChanged(e);
+
+        if (e.Property == ViewProperty)
+        {
+            if (View is GridView gridView)
+            {
+                AddToggleButtonToFirstColumn(gridView);
+            }
+        }
+    }
+
+    private void AddToggleButtonToFirstColumn(GridView gridView)
+    {
+        if (gridView.Columns.Count > 0)
+        {
+            var firstColumn = gridView.Columns[0];
+            firstColumn.CellTemplate = CreateToggleButtonTemplate(firstColumn.CellTemplate);
+        }
+    }
+
+    private DataTemplate CreateToggleButtonTemplate(DataTemplate originalTemplate)
+    {
+        var template = new DataTemplate();
+
+        var stackPanelFactory = new FrameworkElementFactory(typeof(StackPanel));
+        stackPanelFactory.SetValue(StackPanel.OrientationProperty, Orientation.Horizontal);
+
+        var marginMultiBinding = new MultiBinding
```

Comment:
I think these margins are a bit low. I think we want there to be a larger separation between the toggle button and the content (closer to what it is without the GridView). The height of the individual rows when using the GridView is also a bit larger than without. Ideally we would want these to be pretty close to the same default sizes.

With GridView

<img width="436" height="274" alt="Image" src="https://github.com/user-attachments/assets/38d183f9-d13d-42fd-96bf-954829ea376c" />

Without GridView:
<img width="263" height="236" alt="Image" src="https://github.com/user-attachments/assets/0b889b30-8a43-4567-9344-2f4a80d491e5" />